src/traces/table/defaults.js

```diff
@@ -52,6 +52,12 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
     Lib.coerceFont(coerce, 'cells.font', fontDflt);
 
     coerce('header.values');
+    for(var i = 0; i < traceOut.header.values.length; i++) {
+        if(!Array.isArray(traceOut.header.values[i])) {
+            traceOut.header.values[i] = [traceOut.header.values[i]];
+        }
+    }
+
```

Comment:
This makes me vaguely uneasy - having a different structure of the attribute in `data` vs `_fullData`. For example, the workspace looks at `_fullData` for what the current values are in order to construct controls, but then feeds control output back into `data`. It might be fine, but if there's a way to push this farther down, into `data_preparation_helper` perhaps, that would make me more comfortable.

Reply:
Thanks, will do!